rllib/algorithms/alpha_zero/alpha_zero.py

```diff
@@ -221,6 +222,9 @@ def training(
             mcts_config: MCTS specific settings.
             ranked_rewards: Settings for the ranked reward (r2) algorithm
                 from: https://arxiv.org/pdf/1807.01672.pdf
+            num_steps_sampled_before_learning_starts: Number of timesteps to collect
```

Comment:
I assume it is number of environment timesteps, can we make the documentation more explicit here?

Reply:
see my comment below as well.

Reply:
Changed.